sympy/functions/combinatorial/factorials.py

```diff
@@ -829,14 +829,13 @@ class binomial(CombinatorialFunction):
 
     .. math:: \binom{n}{k} = \frac{ff(n, k)}{k!}
 
-    The following identity can be used to handle negative or
-    non-integer values of ``k`` when ``n - k`` is a non-negative
-    integer::
+    The following identity can be used to handle negative integral
+    values of ``k`` when ``n - k`` is a non-negative integer:
 
     .. math:: \binom{n}{k} = \binom{n}{n - k}
 
-    If neither ``k`` nor ``n - k`` is a non-negative integer then
-    the value can be expressed in terms of gamma functions::
+    If either ``n`` nor ``k`` is non-integral then the value is
```

Comment:
```suggestion
    If neither ``n`` nor ``n - k`` is a nonnegative integer then the binomial is
```
You are only forced to use gamma if you `k` and `n-k` aren't nonnegative.